app/controllers/membership_applications_controller.rb

```diff
@@ -264,5 +265,11 @@ def update_error(error_message)
 
   end
 
+  def send_new_membership_application_notice_to_admins(new_membership_application)
+    User.admins.each do |admin|
```

Comment:
This is good.  We _should_ be sending email out to every user that is an admin.  This means that we need to make sure that we use the address for each admin as the @recipient address in the email that is sent out and _not_ use the generic ENV['SHF_MEMBERSHIP_EMAIL'] address.  (The ENV['SHF_MEMBERSHIP_EMAIL'] is used as the return/sender address when sending some emails out.)


1.  Need to have a specification for this (= RSpec test for the mailer )
2. will need to write code in the AdminMailer to handle this

(You need to have ENV['SHF_MEMBERSHIP_EMAIL']  in your .env but while you're just testing, it shouldn't matter what the value is.  The `.env.example` file shows you all of the values that you need (with dummy data).  Use it to compare to your `.env` file to see what keys you might be missing.)

Reply:
@weedySeaDragon ok got it. I will make the recipient address to be the admin to whom the notice is being sent. Will write test for this too.

Reply:
@weedySeaDragon Thanks. Although this one I think is for the admins' emails? 

So is it right in my understanding that `ENV['SHF_MEMBERSHIP_EMAIL`] is "SHF's email" address and would be wrong to send to it for this purpose? 

And that `@recipient_email = record.email if record.respond_to? :email` would be the correct one since this code fetches an admin's email?

And actually I have deleted the private methods in `AdminMailer`, because they are duplicates of those in `CommonMailUtils`?

Although the `#set_recipient_email` in `AdminMailer` is a bit different from the one in `CommonMailUtils` because it sets `ENV['SHF_MEMBERSHIP_EMAIL]` as the recipient, which does not make sense to me since that is not an admin's email?

Reply:
ENV['SHF_MEMBERSHIP_EMAIL] is the email address that people (members and potential members) can send to when they have questions about SHF members.  It is not the 'general SHF email address.' 

It is _not_ a recipient for this email.

The code you have in `send_new_membership_application_notice_to_admins` is the right way to identify and gather the recipients for the email.

> And that @recipient_email = record.email if record.respond_to? :email would be the correct one since this code fetches an admin's email?

I don't know that you mean by this.

>And actually I have deleted the private methods in AdminMailer, because they are duplicates of those in CommonMailUtils?

I don't see this change in your code.  You'll need to push your commits here so that I can see what you mean.